# scDGM
single cell analysis with deep generative model

Some experiments: [clustering](https://htmlpreview.github.io/?https://github.com/jiangyuan2li/scDGM/blob/Jiangyuan/test_clustering.html) of scDeepGMM, [merging](https://htmlpreview.github.io/?https://github.com/jiangyuan2li/scDGM/blob/Jiangyuan/test_merging.html) of scDeepGMM 

References
--------

10X Datasets: https://support.10xgenomics.com/single-cell-gene-expression/datasets

scVI: [Deep generative modeling for single-cell transcriptomics](https://www.nature.com/articles/s41592-018-0229-2) ([github](https://github.com/YosefLab/scVI))

Fair-VAE: [The Variational Fair Autoencoder](https://arxiv.org/pdf/1511.00830.pdf)

GM-VAE: 1. [Deep Unsupervised Clustering with Gaussian Mixture Variational Autoencoders](https://arxiv.org/pdf/1611.02648.pdf)

2. A blog: [GAUSSIAN MIXTURE VAE: LESSONS IN VARIATIONAL INFERENCE, GENERATIVE MODELS, AND DEEP NETS](http://ruishu.io/2016/12/25/gmvae/), ([github](https://github.com/RuiShu/vae-clustering))

GMVAE on single cell ATAC-seq data: [SCALE method for single-cell ATAC-seq analysis via latent feature extraction](https://www.nature.com/articles/s41467-019-12630-7) ([github](https://github.com/jsxlei/SCALE))

scVAE: [scVAE: Variational auto-encoders for single-cell gene expression data](https://www.biorxiv.org/content/10.1101/318295v4) ([github](https://github.com/scvae/scvae))
